app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class QuotesController < ApplicationController
+  def index
+  @keyword = params[:query]
+  @quotes = QuoteFacade.quote_finder(@keyword)
+  end
+end
```

Comment:
This looks good. You'll also need to show the total number of results on the results page, and that comes from the response from the endpoint you're consuming.